test/tap/search.esearch.js

```diff
@@ -0,0 +1,192 @@
+var common = require('../common-tap.js')
```

Comment:
That's just me being inconsistent. I've been using the former with pacote and cacache and it just feels nicer for hierarchies of tests in a single flat dir